kyuubi-common/src/test/scala/org/apache/kyuubi/operation/DeltaRowLevelOperationTests.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.operation
+
+import org.apache.kyuubi.DeltaSuiteMixin
+
+trait DeltaRowLevelOperationTests extends HiveJDBCTestHelper with DeltaSuiteMixin {
```

Comment:
how about we rename it to DML and bring more cases in?

Reply:
Row level operation is the spark offical term introduced in https://issues.apache.org/jira/browse/SPARK-35801, I'm planning to add other cases later in `TimeTravelTests` `SecondaryIndexTests` etc to reflect different capbilities in different table format

Reply:
The complexity of mixins brings more overhead of maintenance. I have never experienced using traits for classification, not abstraction

Reply:
what's the behavior of iceberg or delta ?

Reply:
Any suggestion for classification expect traits in scalatest? @yaooqinn

Reply:
> what's the behavior of iceberg or delta ?

I don't get your point.

Reply:
BTW, I explained the idea in #1287, if we reject to use traits for classification, we need to reconsider the test classes hierarchical